### ISSUE TITLE
Pull to refresh not working inside elements with height 100%

### ISSUE THREAD

**Issue by matiastucci** [framework7io/framework7#2944]
* Framework7 version: 3.6.5
* Platform and Target: browser
* Live Link or JSFiddle: https://jsbin.com/wojuwiture/edit?html,js,output

### Describe the bug
Pull to refresh is not working when I try to pull elements inside this structure:

```html
<div class="page-content ptr-content">
  <div class="ptr-preloader">
      <div class="preloader"></div>
      <div class="ptr-arrow"></div>
  </div>
  <div class="bar" style="height: 100%">
      <div class="foo" style="height: 100%">
        // HERE
      </div>
  </div>
</div>
```

I was using v2 before and it was working fine. I created other JS bin link to show that: https://jsbin.com/sebufohofa/edit?html,output.

### To Reproduce
1. Go to the JS bin link: https://jsbin.com/wojuwiture/edit?html,js,output
2. Try to pull elements

### Expected behavior
See the PTR

### Actual Behavior
Doesn't pull

**Comment by matiastucci** [comment]
Actually, that doesn't fix my example.

`e.target !== el` evaluates to `true` and then `targetIsScrollable` is set to `true`. Maybe we can have a CSS class like `.not-scrollable` or whatever and check that too what do you think?

**Comment by nolimits4web** [comment]
Can you show JSFiddle where this fix won’t work? I checked with your layout like in your fiddle and it works correct

**Comment by matiastucci** [comment]
Sure: https://jsbin.com/deyutedofi/edit?html,output

I just copied/paste https://framework7.io/packages/core/js/framework7.js and added the `if` of https://github.com/framework7io/framework7/commit/a8a2de7aca4c5808f3c8189d91fb728cf48bcea6

Also, in my real app the `targetEl.scrollHeight > targetEl.offsetHeight` check sometimes is `true` but is not a real scrollable item. What do you think of checking an specific class name?

**Comment by matiastucci** [comment]
Cool, thanks. Is it possible to have this fix in v3.6.x?

**Comment by nolimits4web** [comment]
Yep, will land in next update on a days
